docs/commands/oras_cp.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: oras cp
```

Comment:
Is this `oras cp` doc overlapped with oras copy? https://oras.land/docs/commands/oras_copy?

Reply:
`copy` is an alias

Reply:
Fixed.